app/lib/audit/models.dart

```diff
@@ -139,11 +151,57 @@ class AuditLogRecord extends db.ExpandoModel<String> {
       ..packageVersions = ['$package/$version']
       ..publishers = [];
   }
+
+  factory AuditLogRecord.packageTransferred({
+    @required User user,
+    @required String package,
+    @required String fromPublisherId,
+    @required String toPublisherId,
+  }) {
+    return AuditLogRecord.init()
+      ..kind = AuditLogRecordKind.packageTransferred
+      ..agent = user.userId
+      ..summary = [
+        'Package `$package` ',
+        if (fromPublisherId != null) 'from `$fromPublisherId` ',
+        'was transferred to publisher `$toPublisherId` ',
+        'by `${user.email}`.',
+      ].join()
+      ..data = {
+        'package': package,
+        if (fromPublisherId != null) 'fromPublisherId': fromPublisherId,
+        'toPublisherId': toPublisherId,
+        'user': user.email,
+      }
+      ..users = [user.userId]
+      ..packages = [package]
+      ..packageVersions = []
+      ..publishers = [
+        if (fromPublisherId != null) fromPublisherId,
+        toPublisherId,
+      ];
+  }
 }
 
 abstract class AuditLogRecordKind {
   /// Event that a package was published.
   ///
   /// This can be an entirely new package or just a new version to an existing package.
   static const packagePublished = 'package-published';
+
+  /// Event that a package has transferred to a (new) publisher.
+  static const packageTransferred = 'package-transferred';
+
+  // TODO: implement further models
```

Comment:
```suggestion
  // TODO: implement further kinds
```

We're not adding new entity kinds or anything :D